### ISSUE TITLE
22. Tokenized List Test 20 fails seemingly incorrectly

### ISSUE THREAD

**Issue by knorke3** [firasdib/Regex101#2204]
### Task
22. Tokenized List

### Bug Description
Test 20 Fails the regex `redacted` with the following message:
**redacted**
while using the same regex on the given example in the unit tests using PCRE2 succeeds:
**redacted**

While experimenting, i noticed that removing `\G` from the front of the regex causes it on test 19 rather than 20 with the same message (except for slight formatting differences).

### Reproduction steps
- Paste `redacted` into the solution for Task 22
- The task fails on test 20
- Paste the same regex into a Unit Test with PCRE2 against `"this_item,should,not,match`
- The Unit Test succeeds

### Expected Outcome
The Unit Test and the Tests in the Task should result in the same outcome (or, if it happens due to the Tasks using a different flavor, the flavor used should be specified somewhere).

### Browser
Firefox 121.0 (64 Bit)

### OS
Windows 11

**Comment by firasdib** [comment]
I removed your solution to not spoil it for others. If you please could join IRC/Discord and discuss the matter there instead, that would be great.

**Comment by knorke3** [comment]
Good point - didn't consider it as i hadn't gotten the "solved" on the task yet. Will join as soon as i have access to my discord.

**Comment by jtmdrums** [comment]
Agreed that this question is bugged for Test 20, I'm running the exact same regex that I have as my answer in the debugger and it performs as expected against that test input. Would really like to move past this question

**Comment by Preasured** [comment]
@firasdib I'm helping someone through a a similar issue but OP's question wasn't resolved on the RegEx 101 Discord (under "bugs"). Is there a way we can get a sample set of test strings that we can use to resolve false positives?
